Source/Frontend/StandaloneRTC/StandaloneRTC.csproj

```diff
@@ -28,29 +17,24 @@
     <IsWebBootstrapper>false</IsWebBootstrapper>
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <UseWindowsForms>true</UseWindowsForms>
+    <!-- Use Build\ instead of Build\net471\ for output -->
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <!-- RuntimeIdentifier needed for dolphin -->
+    <RuntimeIdentifiers>win7-x64</RuntimeIdentifiers>
+    <!-- Avoid MissingManifestResourceException  errors, see https://github.com/dotnet/ef6/issues/1258#issuecomment-531355034 -->
+    <EmbeddedResourceUseDependentUponConvention>true</EmbeddedResourceUseDependentUponConvention>
```

Comment:
This workaround may need to go in more projects, but I don't want to over-include it.

Reply:
The StandaloneRTC project really shouldn't be included in the SLNs of the other projects so if it's included in Dolphin and preventing it from building without the RuntimeIdentifiers, we should remove that.  

On the second one, could you also add this as part of the comment? https://docs.microsoft.com/en-us/dotnet/core/compatibility/msbuild#resource-manifest-file-name-change

Reply:
I opened #243 to address the first issue, but I think for this PR it's out of scope.

I've updated the comment on the second one to have a bunch of links for context.